dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -34,6 +34,17 @@ def post_ui_tip_dismissed
     render status: 200, json: {set: property_name}
   end
 
+  # POST /api/v1/users/<user_id>/accept_data_transfer_agreement
```

Comment:
What prevents me from accepting the agreement on behalf of other users?  Do we really need the `user_id` in the URL, or could we always apply this call to the `current_user`?